docs/LICENSE-MIT

```diff
@@ -18,4 +16,4 @@ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
 AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
```

Comment:
Is this diff just a new trailing newline? I assume so - GitHub mobile doesn't make it that clear

Reply:
Almost, the change is that trailing whitespace was added.
This needs to be reverted to satisfy the markdownlint ci check.